ansible_collections/arista/avd/plugins/modules/eos_validate_state_runner.py

```diff
@@ -12,64 +12,80 @@
 module: eos_validate_state_runner
 version_added: "4.4.0"
 author: Arista Ansible Team (@aristanetworks)
-short_description: Leverage ANTA for eos_validate_state role
+short_description: Utilizes ANTA for eos_validate_state role validation
 description:
   - The C(arista.avd.eos_validate_state_runner) module is an Ansible Action Plugin leveraging the ANTA test
     framework to validate that the generated structured configurations by AVD are applied to the devices and
     that the deployed network is working correctly.
-  - This plugin expects that the structued_configs of each device is present in hostvars, otherwise no test will be generated.
+  - This plugin requires that structured_configs for each device be present in hostvars; otherwise, no tests will be executed.
```

Comment:
technically generated is more correct - you can generate the tests without executing them if you run in check mode.
```suggestion
  - This plugin requires that structured_configs for each device be present in hostvars; otherwise, no tests will be generated.
```

Reply:
Fixed in 68479d7a6549495d8a3a4e5ce970b358e51ea5bc